shell/packages/sandstorm-ui-grainlist/grainlist.html

```diff
@@ -3,7 +3,10 @@
     <div class="non-usage-info">
     <h1>Grains</h1>
     <div class="search-row">
-      <img title="Search" class="search-icon" src="/search.svg"><input class="search-bar" type="text" placeholder="search" value="{{ searchText }}">
+      <label>
+        <span title="Search" class="search-icon"></span>
```

Comment:
Yes.  The structure matches the first example on MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

If you were asking about `title`: `title` is a Global attribute, which may be specified on all HTML elements: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes

(for bonus points, it's a `<span>` rather than a `<div>` because `<div>`s aren't allowed as children of `<label>`s)